As reported elsewhere[1], enabling Unicode fonts on Cygwin (which doesn't
have proper Unicode support) causes only boxes to be rendered instead of 
characters.  Using X_LOCALE does not help.

[1] http://www.stokebloke.com/cygwin/index.php

--- origsrc/blackbox-0.70.1/lib/Unicode.cc	2005-01-03 03:42:53.000000000 -0600
+++ src/blackbox-0.70.1/lib/Unicode.cc	2008-11-23 16:22:04.539546800 -0600
@@ -158,6 +158,9 @@
 } // namespace bt
 
 bool bt::hasUnicode() {
+#ifdef __CYGWIN__
+  return false;
+#endif
   static bool has_unicode = true;
   static bool done = false;
 
